src/python/WMCore/WMBS/CreateWMBSBase.py

```diff
@@ -278,15 +278,15 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE TABLE wmbs_job (
              id           INTEGER       PRIMARY KEY AUTO_INCREMENT,
              jobgroup     INTEGER       NOT NULL,
-             name         VARCHAR(255),
+             name         VARCHAR(70),
              state        INTEGER       NOT NULL,
              state_time   INTEGER       NOT NULL,
              retry_count  INTEGER       DEFAULT 0,
              couch_record VARCHAR(255),
              location     INTEGER,
              outcome      INTEGER       DEFAULT 0,
-             cache_dir    VARCHAR(767)  DEFAULT 'None',
-             fwjr_path    VARCHAR(767),
+             cache_dir    VARCHAR(70)  DEFAULT 'None',
+             fwjr_path    VARCHAR(70),
```

Comment:
Just pointing out you can't commit any of these short column length because it'll break everything. If this is just an intermediate stage to get some tests to run, fine. But we can't forget about changing this back to longer columns.